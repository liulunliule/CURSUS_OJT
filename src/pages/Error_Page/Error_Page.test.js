import React from 'react';
import { render } from '@testing-library/react';
import ErrorPage from '../Error_Page';

test('renders error page component correctly', () => {
  const { asFragment } = render(<ErrorPage />);
  expect(asFragment()).toMatchSnapshot();
});
