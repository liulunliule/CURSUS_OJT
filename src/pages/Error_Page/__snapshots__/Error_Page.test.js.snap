// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders error page component correctly 1`] = `
<DocumentFragment>
  <div
    class="error-page error-page-wrapper"
  >
    <div
      class="container"
    >
      <div
        class="row"
      >
        <div
          class="col-md-12"
        >
          <div
            class="cmtk_group"
          >
            <div
              class="ct-logo"
            >
              <a
                href="index.html"
              >
                <img
                  alt=""
                  src="logo_dark.svg"
                />
              </a>
            </div>
            <div
              class="cmtk_dt"
            >
              <h1
                class="title_404"
              >
                404
              </h1>
              <h4
                class="thnk_title1"
              >
                The page you were looking for could not be found.
              </h4>
              <a
                class="bk_btn"
                href="/"
              >
                Go To Homepage
              </a>
            </div>
            <div
              class="tc_footer_main"
            >
              <div
                class="tc_footer_left"
              >
                <ul>
                  <li>
                    <a
                      href="about_us.html"
                    >
                      About
                    </a>
                  </li>
                  <li>
                    <a
                      href="press.html"
                    >
                      Press
                    </a>
                  </li>
                  <li>
                    <a
                      href="contact_us.html"
                    >
                      Contact Us
                    </a>
                  </li>
                  <li>
                    <a
                      href="coming_soon.html"
                    >
                      Advertise
                    </a>
                  </li>
                  <li>
                    <a
                      href="coming_soon.html"
                    >
                      Developers
                    </a>
                  </li>
                  <li>
                    <a
                      href="terms_of_use.html"
                    >
                      Copyright
                    </a>
                  </li>
                  <li>
                    <a
                      href="terms_of_use.html"
                    >
                      Privacy Policy
                    </a>
                  </li>
                  <li>
                    <a
                      href="terms_of_use.html"
                    >
                      Terms
                    </a>
                  </li>
                </ul>
              </div>
              <div
                class="tc_footer_right"
              >
                <p>
                  Â© 2024 
                  <strong>
                    Cursus
                  </strong>
                  . All Rights Reserved.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <script
    src="js/jquery-3.7.1.min.js"
  />
  <script
    src="vendor/bootstrap/js/bootstrap.bundle.min.js"
  />
  <script
    src="vendor/OwlCarousel/owl.carousel.js"
  />
  <script
    src="vendor/bootstrap-select/docs/docs/dist/js/bootstrap-select.js"
  />
  <script
    src="vendor/semantic/semantic.min.js"
  />
  <script
    src="js/custom.js"
  />
</DocumentFragment>
`;
